packages/Communication/core/src/BaseProvider.ts

```diff
@@ -216,7 +216,7 @@ export class ProcessedMessage extends Message {
  * MessageResult class, holds information and functionality specific to a single message result
  */
 export class MessageResult {
-    public Run: CommunicationRunEntity;
+    public Run?: CommunicationRunEntity;
```

Comment:
@AN-BC I made this optional to get the build working. Seemed reasonable based on the fact that the SendGridProvider implementation returns results without `Run` defined, but let me know if there's something else you had in mind.